SekoiaIO/endpoint/_meta/smart-descriptions.json

```diff
@@ -3,6 +3,10 @@
         "value": "Hostname: {host.hostname} - Source: {event.provider} - EventID: {event.code}",
         "conditions": []
     },
+    {
+        "value": "Action: {event.action} - Hostname: {host.hostname}",
+        "conditions": []
```

Comment:
As best practices for smart-description, we set the existence of the fields used in the value as conditions.

Reply:
Thanks for the review, I will make the change then